gunicorn/config.dev.py

```diff
@@ -0,0 +1,239 @@
+import datetime
+
+import pytz
+from gunicorn.glogging import Logger
+
+# Gunicorn configuration file.
+#
+# Server socket
+#
+#   bind - The socket to bind.
+#
+#       A string of the form: 'HOST', 'HOST:PORT', 'unix:PATH'.
+#       An IP is a valid HOST.
+#
+#   backlog - The number of pending connections. This refers
+#       to the number of clients that can be waiting to be
+#       served. Exceeding this number results in the client
+#       getting an error when attempting to connect. It should
+#       only affect servers under significant load.
+#
+#       Must be a positive integer. Generally set in the 64-2048
+#       range.
+#
+
+bind = "127.0.0.1:5000"
```

Comment:
If this is running on PaaS, shouldn't port be 8080?
Does this meaningfully differ from test? If not can we delete it?

Reply:
Ah sorry, hangover from local testing of dev - removed in pref for combined dev_test config file